docs/src/styles/custom-css.css

```diff
@@ -50,6 +53,18 @@
   margin: 0px 2px;
 }
 
+a.action, 
+.article-footer > a, 
+.action-footer > a.action-button {
```

Comment:
- `a.action` are all landing-page start buttons, get started, go to latest challenge ...
- `.article-footer > a` explains itself 😄 
- the `.action-footer` is in the right sidebar on a content page